test/test_DFT_deconv.py

```diff
@@ -0,0 +1,101 @@
+"""Test PyDynamic.uncertainty.propagate_DFT.DFT_deconv"""
+import numpy as np
+import scipy.stats as stats
+from hypothesis import given, settings
+from hypothesis.strategies import data
+from numpy.testing import assert_allclose
+
+from PyDynamic.uncertainty.propagate_DFT import DFT_deconv
+from .conftest import (
+    hypothesis_covariance_matrix_for_complex_vectors,
+    nonzero_complex_vector,
+    nonzero_float_vector,
+    two_to_the_k,
+)
+
+
+@given(data(), two_to_the_k(min_k=2, max_k=4))
+@settings(deadline=None)
+def test_dft_deconv(
+    hypothesis,
+    n,
+):
+    covariance_scale_minimizer = 1e-3
+    y_complex = hypothesis.draw(nonzero_complex_vector(length=n))
+    y = np.r_[y_complex.real, y_complex.imag]
+    uy = covariance_scale_minimizer * hypothesis.draw(
+        hypothesis_covariance_matrix_for_complex_vectors(n)
+    )
+    h_complex = hypothesis.draw(nonzero_complex_vector(length=n))
+    h = np.r_[h_complex.real, h_complex.imag]
+    uh = covariance_scale_minimizer * hypothesis.draw(
+        hypothesis_covariance_matrix_for_complex_vectors(n)
+    )
+    x_deconv, u_deconv = DFT_deconv(H=h, Y=y, UH=uh, UY=uy)
+    n_monte_carlo_runs = 40000
+    y_mc = stats.multivariate_normal.rvs(mean=y, cov=uy, size=n_monte_carlo_runs)
+    h_mc = stats.multivariate_normal.rvs(mean=h, cov=uh, size=n_monte_carlo_runs)
+    real_complex_divider_index = 2 * n // 2
```

Comment:
I am curious: In which corner cases `n != real_complex_divider_index` ?

Reply:
It will be different for odd n. I took this snippet equivalently from the original functions indexing: https://github.com/PTB-M4D/PyDynamic/blob/befb588c45c29a6aa746379eddc5b9653acdd653/PyDynamic/uncertainty/propagate_DFT.py#L971.

Reply:
I love your humble question: "In which 'corner cases' do we have 2/2 != 1?" Made my day!

Reply:
Resolved with a12d42d.